oops/backplane/ring.py

```diff
@@ -654,21 +654,22 @@ def ring_radial_resolution(self, event_key):
     return self.register_backplane(key, resolution)
 
 #===============================================================================
-def ring_angular_resolution(self, event_key):
+def ring_angular_resolution(self, event_key, format="deg"):
     """Projected angular resolution in radians/pixel at the ring intercept.
 
     Input:
         event_key       key defining the ring surface event. Alternatively, a
                         ring_radius or radial_mode backplane key, in which case
                         this backplane inherits the mask of the given backplane
                         array.
+        format          longitude representation; "deg" or "km"
```

Comment:
Please don't call this "format", as that is a reserved word. Also, "deg" is not right because the values are radians. If you plan to stick with a string value options for the input, rather than a boolean, be sure to add a check at the beginning and raise ValueError if the input is not one of the valid options.

Reply:
changed to "units"